### PR TITLE
Created Scripts to help with necessary configuration.

### PR DESCRIPTION
**Pull request by diminishedprime**
Since first-time users will need to configure:

+ a development `MEASUREMENT_ID`
+ a production `MEASURUMENT_ID`
+ oauth info
+ firebase info

but we don't want any of this data to be version controlled/exposed trivially, this PR adds in a set of scripts that allow for these values to be configured in a `runtime.json` file. If the file doesn't exist, inquirer is used to prompt the user for values. It adds a bit of time to the scripts that it's involved in (namely `yarn build` & `yarn develop`), but these scripts are already somewhat slow. 

This change also makes it straightforward for us to add in new configuration in the future since the prompt is always run (and skipped if no new data is needed). 

In the future, if we want to speed up these scripts a bit, we can remove re running tsc every time & committing the resulting building directory into the repo. I'm not doing this for now because it adds complexity if the scripts & build directory go out of sync.

### CODE REVIEW

File: to-gatsby/lib/scripts/check-config.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const configQuestions = (
+  filter: ConfigQuestionFilter
+): inquirer.QuestionCollection<RuntimeJson> => {
+  return [
+    {
+      name: "gaMeasurementId",
+      // TODO - Nice to have, list the user's available properties. Would
+      // require the user to authenticate first to make the request.
+      type: "input",
+      message: "Measurement ID for production: ",
```
Comment (pfrisella) [inline to-gatsby/lib/scripts/check-config.ts:40]: Google Analytics Measurement ID

File: to-gatsby/lib/scripts/check-config.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      when: () => {
+        return filter.askAll || filter.gaMeasurementId === undefined
+      },
+    },
+    {
+      name: "gaMeasurementIdDev",
+      // TODO - Nice to have, list the user's available properties. Would
+      // require the user to authenticate first to make the request.
+      type: "input",
+      message: "Measurement ID for development: ",
```
Comment (pfrisella) [inline to-gatsby/lib/scripts/check-config.ts:51]: Google Analytics Measurement ID

File: to-gatsby/lib/scripts/check-config.ts (hunk 1 above)
Comment (diminishedprime) [reply to-gatsby/lib/scripts/check-config.ts]: Done.

File: to-gatsby/lib/scripts/check-config.ts (hunk 2 above)
Comment (diminishedprime) [reply to-gatsby/lib/scripts/check-config.ts]: Done

**Pull request merged**